### PR TITLE
feat(form): add new question type action button

### PR DESCRIPTION
**Pull request by anehx**
Add a new question type for action buttons that render a button for completing or skipping the work item related to the current document.

Backend implementation: https://github.com/projectcaluma/caluma/pull/1607

### CODE REVIEW

File: packages/form/addon/components/cf-field/input/action-button.hbs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{{#unless @disabled}}
+  <DocumentValidity
```
Comment (Yelinz) [inline packages/form/addon/components/cf-field/input/action-button.hbs:2]: Might want to add an `onInvalid` notification?

**Review by Yelinz — APPROVED** [review]
lgtm

File: packages/form/addon/components/cf-field/input/action-button.hbs (hunk 1 above)
Comment (anehx) [reply packages/form/addon/components/cf-field/input/action-button.hbs:2]: No, the button will be disabled if it's invalid.

**Pull request merged**